documentation/course/e2/README.md

```diff
@@ -269,44 +299,125 @@ TODO @mlodybercik: porównanie z on-premise i najlepiej innymi chmurami
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th><i>on-premise</i></th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Pełna kontrola nad danymi</li>
+        <li>Brak zależności od dostawcy usług</li>
+        <li>Niski koszt przy wysokim obciążeniu</li>
+        <li>Niskie opóźnienia</li>
+        <li>Brak ukrytych kosztów</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Wysoki koszt początkowy</li>
+        <li>Wysoki koszt utrzymania</li>
+        <li>Brak możliwości skalowania w dół</li>
+        <li>Brak możliwości szybkiej modernizacji</li>
+        <li>Płacenie za utrzymanie całej infrastruktury nawet gdy korzystamy z jej części</li>
       </ul>
     </td>
   </tr>
   <tr>
-    <th>Rozwiązanie 2</th>
+    <th>Chmura</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Brak konieczności inwestowania w sprzęt</li>
+        <li>Możliwość szybkiego skalowania</li>
+        <li>Wysoka niezawodność</li>
+        <li>Bezpieczeństwo danych</li>
+        <li>Wysoka dostępność usług</li>
+        <li>Możliwość szybkiej modernizacji</li>
+        <li>Możliwość dostosowania infrastruktury do indywidualnych potrzeb</li>
+        <li>Płacisz tylko za to co używasz</li>
+        <li>Duży wybór dostawców usług chmurowych</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Brak pełnej kontroli nad danymi</li>
+        <li>Zależność od dostawcy usług</li>
+        <li>Stosunkowo wysoki koszt przy wysokim obciążeniu</li>
+        <li>Stosunkowo wysokie opóźnienia</li>
```

Comment:
Again, in what sense?

Reply:
https://github.com/PWR-ACS-SE-24/SoftwareSystemDesign/pull/24/files#r1885825575